build.sbt

```diff
@@ -54,9 +54,9 @@ libraryDependencies ++= Seq(
   "javax.servlet"                   %  "javax.servlet-api"            % "3.1.0"          % "provided",
   "junit"                           %  "junit"                        % "4.12"           % "test",
   "org.scalatra"                    %% "scalatra-scalatest"           % ScalatraVersion  % "test",
-  "org.mockito"                     %  "mockito-core"                 % "2.7.16"         % "test",
+  "org.mockito"                     %  "mockito-core"                 % "2.7.22"         % "test",
   "com.wix"                         %  "wix-embedded-mysql"           % "2.1.4"          % "test",
-  "ru.yandex.qatools.embed"         %  "postgresql-embedded"          % "1.14"           % "test"
+  "ru.yandex.qatools.embed"         %  "postgresql-embedded"          % "1.23"           % "test"
```

Comment:
Java 9 should work now (at least now). Please merge.

> Some users are already trying to use GitBucket with Java 9.

I understand that, but IMHO this does not belong in the master branch (maybe a Java 9 branch until Travis and all other support Java 9 officially too).